app/views/promotion/landingPage_incentive.scala.html

```diff
@@ -30,15 +36,17 @@
             <div class="page-container gs-container gs-container--slim page-slice">
                 <div class="digipack">
                     <div class="grid grid--3up grid--flex">
-                        <div class="grid__item grid__item--flex-2-columns">
-                            <div class="digipack__shots">
-                                @fragments.common.roundel(promotion.roundelHtml, Seq("roundel--raised"))
-                            </div>
+                        <div class="grid__item grid__item--flex-2-columns digipack__shots">
+                            @fragments.common.roundel(promotion.roundelHtml, Seq("roundel--raised"))
                         </div>
                         <div class="grid__item digipack__benefits">
-                            <p class="digipack__benefit">Experience your newspaper in full, on your tablet with the Guardian & Observer daily edition</p>
-                            <p class="digipack__benefit">Enjoy an advert-free experience on The Guardian's app with access to our daily crosswords and exclusive content</p>
-                            @subscribeNow(edition, catalog.digipackMonthly.priceGBP, promoCode)
+                            <div class="digipack__benefits--height">
```

Comment:
in BEM the idea of modifier classes is that they are applied to the same element as the base class
```html
<div class="digipack__benefits digipack__benefits--height">
...
</div>